README.md

```
@@ -2,27 +1,237 @@
+-   Press F5 to run the API (alternatively, click **▶ Start** button in the top toolbar)
+
+### Setting Up Frontend
+
+-   Go back to the terminal, navigate to the `frontend/finance-tracker` directory and install dependencies.
+
+```bash
+cd frontend/finance-tracker
+npm i
+```
+
+### Generate Auth Secret:
```

markdownlint-cli2 (0.17.2)

Trailing punctuation in heading
Punctuation: ':'

(MD026, no-trailing-punctuation)